### ISSUE TITLE
Double Click on restaurant pos

### ISSUE THREAD

**Issue by webbosolutions** [quantumbitcore/erpnext-restaurant#9]
is there any solution to change double click on pos cashier like new order , pay button to be only one click to be easy for the cashier user
Thanks

**Comment by quantumbitcore** [comment]
Yes that can be done, I did it so that no orders are created by mistake.  it can be removed directly from the code or it can be made configurable. 

In the onClick event remove DOUBLE_CLICK constant.

**Comment by quantumbitcore** [comment]
An update is now available for this feature.

**Comment by webbosolutions** [comment]
Thanks Thanks
Did Updates Finished Please ?
Best Regards

**Comment by quantumbitcore** [comment]
Yeah, is ready.
